php/pantheon/checks/plugins.php

```diff
@@ -69,35 +69,99 @@ public function run() {
 	}
 
 	/**
-	* Checks the plugin slug against the vulnerability db
+	 * Checks the plugin slug against the vulnerability db
+	 * @param $plugin_slug string (required) string representing the plugin slug
+	 *
+	 * @return array containing vulnerability info or false
+	 */
+	protected function getPluginVulnerability( $plugin_slug )
+    {
+		// Get the vulnerability API token from the platform
+		$wpvulndb_api_token = getenv('PANTHEON_WPVULNDB_API_TOKEN');
+
+		// Throw an exception if there is no token
+		if( false === $wpvulndb_api_token || empty( $wpvulndb_api_token ) ) {
+			throw new \Exception('No WP Vulnerability DB API Token. Please ensure the PANTHEON_WPVULNDB_API_TOKEN environment variable is set');
```

Comment:
@greg-1-anderson I'm not sure if throwing an exception is what we want to do here. The alternative is to not complete the check and fail silently, which is not desired either.

Reply:
I am +1 on total catastrophic failure when there is no API token set.

Well, throwing an exception is probably sufficient. No need to call `exit` or erase the hard drive or anything.